### ISSUE TITLE
Breaking - error: failed to select a version for `snow`

### ISSUE THREAD

**Issue by balqaasem** [setheum/setheum#548]
```
error: failed to select a version for `snow`.
    ... required by package `libp2p-noise v0.30.0`
    ... which is depended on by `libp2p v0.37.1`
    ... which is depended on by `sc-cli v0.10.0-dev (https://github.com/paritytech/substrate?tag=monthly-2021-08#4d28ebeb)`
    ... which is depended on by `frame-benchmarking-cli v4.0.0-dev (https://github.com/paritytech/substrate?tag=monthly-2021-08#4d28ebeb)`
    ... which is depended on by `setheum-cli v0.9.1 (/Users/muhammad-jibril/Documents/GitHub/Setheum/node/cli)`
    ... which is depended on by `setheum v0.9.1 (/Users/muhammad-jibril/Documents/GitHub/Setheum/node)`
versions that meet the requirements `^0.7.1` are: 0.7.2, 0.7.1

all possible versions conflict with previously selected packages.

  previously selected package `snow v0.7.1`
    ... which is depended on by `libp2p-noise v0.30.0`
    ... which is depended on by `libp2p v0.37.1`
    ... which is depended on by `sc-cli v0.10.0-dev (https://github.com/paritytech/substrate?tag=monthly-2021-08#4d28ebeb)`
    ... which is depended on by `frame-benchmarking-cli v4.0.0-dev (https://github.com/paritytech/substrate?tag=monthly-2021-08#4d28ebeb)`
    ... which is depended on by `setheum-cli v0.9.1 (/Users/muhammad-jibril/Documents/GitHub/Setheum/node/cli)`
    ... which is depended on by `setheum v0.9.1 (/Users/muhammad-jibril/Documents/GitHub/Setheum/node)`
```

**Comment by balqaasem** [comment]
Bump to resolve this: https://github.com/paritytech/substrate/pull/9391

**Comment by balqaasem** [comment]
Closes in #554
